images/source/Dockerfile

```diff
@@ -8,4 +8,4 @@
 FROM centos:7
 
 ADD origin-local-release.repo /etc/yum.repos.d/
-ADD centos-paas-sig-openshift-future.repo /etc/yum.repos.d/
+ADD centos-paas-sig-openshift-origin36.repo /etc/yum.repos.d/
```

Comment:
I don't really understand this commit -- is this just to bring in stuff without rebasing?